src/js/letters/containers/AddressSection.jsx

```diff
@@ -144,14 +144,22 @@ export class AddressSection extends React.Component {
     let type = ADDRESS_TYPES.domestic;
     if (!['USA', 'US'].includes(address.country)) {
       type = ADDRESS_TYPES.international;
-    } else if (address.militaryStateCode) {
-      // TODO: Make sure we clear this out if a state code is selected
+    } else if (['AE', 'AA', 'AP'].includes(address.state)) {
+      // Are these ^^ constants anywhere?
       type = ADDRESS_TYPES.military;
     }
 
     return Object.assign({}, address, { type });
   }
 
+  handleCancel = () => {
```

Comment:
Thought this was already in master? Hm.

Reply:
Oh I guess this came in from the merge, forgot this wasn't rebased in.

Reply:
Yeah, it was just part of a merge conflict, so here it is...again. Yay, reruns!